forge-gui/res/cardsfolder/upcoming/dark_apostle.txt

```diff
@@ -0,0 +1,8 @@
+Name:Dark Apostle
+ManaCost:3 R
+Types:Creature Astartes Warlock
+PT:3/3
+A:AB$ Effect | PrecostDesc$ :Gift of Chaos — | Cost$ 3 T |  StaticAbilities$ GrantCascade | SpellDescription$ The next noncreature spell you cast this turn has cascade. (When you cast that spell, exile cards from the top of your library until you exile a nonland card that costs less. You may cast it without paying its mana cost. Put the exiled cards on the bottom of your library in a random order.)
+SVar:GrantCascade:Mode$ Continuous | EffectZone$ Command | OneOff$ True | Affected$ Card.nonCreature+YouCtrl+YouOwn | AffectedZone$ Stack | AddKeyword$ Cascade | Description$ As you cast spells from your hand this turn, they gain cascade.
```

Comment:
YouOwn + Description is wrong

Also OneOff$ is only for triggers, you're gonna have to find another way to clean up for the next part